src/docs/user/references/VariablesReference.adoc

```diff
@@ -222,7 +222,7 @@ ProgressFile.setProgress(variables.get("PA_TASK_PROGRESS_FILE"), 50);
 | all scripts
 | -
 
-| *Cloud Automation Rest URL*. The REST url of the cloud automation service. The public REST url is also available (if the setting pa.cloud-automation.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
+| *Service Automation Rest URL*. The REST url of the service automation. The public REST url is also available (if the setting pa.cloud-automation.rest.public.url is defined in `PROACTIVE_HOME/config/scheduler/settings.ini`)
```

Comment:
Also for Rest and REST, we should use one name in the whole doc, and I suggest **REST**